src/common/gui/CSurgeSlider.cpp

```diff
@@ -791,7 +791,16 @@ bool CSurgeSlider::onWheel(const VSTGUI::CPoint& where, const float &distance, c
       wheelInitiatedEdit = true;
       beginEdit();
    }
-   *edit_value += rate * distance;
+   
+   if (isStepped == true && !(buttons & kControl))
+   {
+      *edit_value += distance / (intRange);
```

Comment:
One tiny change. If intRange is 0, on some systems, this can cause an exception which would crash the DAW and the plugin. intRange shouldn't be zero but it's not guaranteed. I would make this `if( intRange && isStepped = true && !(buttons&kControl))`

Reply:
also isStepped == true can be replaced with just isStepped

Reply:
Didn't think about that. Good idea.